source/documentation/guidance/php-database.md

```diff
@@ -0,0 +1,160 @@
+## Connect a PHP app to PostgreSQL and MySQL
+
+If your PHP app uses PostgreSQL or MySQL, it must connect to those databases securely using SSL. 
+
+> These instructions assume that your app uses the PHP Data Objects (PDO) library to connect to either a MySQL or PostgreSQL backing service database.
+
+You configure your app to enable this secure connection by inserting the following code into the `config.ini` file located within `.bp-config/php/php.ini.d/`:
+
+```
+extension=pdo.so
+extension=pdo_mysql.so OR pdo_pgsql.so
+extension=openssl.so
+```
+
+You should use this method even if you have previously defined PHP extensions in the `.bp-config/options.json` buildpack config file. 
```

Comment:
Does it make sense to be explicit that this is "instead of" and not "as well as"?

Reply:
yes that's a good point